### PR TITLE
Notification-modal_CU-mfd024

### PR DESCRIPTION
**Pull request by a-wilkes**

### CODE REVIEW

**Comment by a-wilkes** [comment]
Task linked: [CU-mfd024 Notification modal](https://app.clickup.com/t/mfd024)

**Comment by lpboucher** [comment]
Task linked: [CU-mfd024 Notification modal](https://app.clickup.com/t/mfd024)

File: src/app/modules/products/components/featured-product-card/featured-product-card.component.html [hunk 1]
```diff
@@ -10,7 +10,7 @@ <h2 class="featured-product-card-body-title">{{product.name}}</h2>
         <p class="featured-product-card-body-description" *ngFor="let description of product.displayedDescription">{{description}}</p>
         <div class="featured-product-card-body-price-container">
             <div class="featured-product-card-body-price">{{product.price}}</div>
-            <button class="featured-product-card-body-get-notified" (click)="onClick()">Get notified</button>
+            <button class="featured-product-card-body-get-notified" (click)="showModal()">Get notified</button>
```
Comment (lpboucher) [inline src/app/modules/products/components/featured-product-card/featured-product-card.component.html]: the method name should always refer to the user action. Here, we should use onGetNotifiedClicked

File: src/app/modules/products/components/get-notified-confirmation-modal/get-notified-confirmation-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { Component, ViewChild } from "@angular/core";
+import { ModalComponent } from "@shared/components/modal/modal.component";
+
+@Component({
+    selector: "app-get-notified-confirmation-modal",
+    templateUrl: "./get-notified-confirmation-modal.component.html",
+    styleUrls: ["./get-notified-confirmation-modal.component.scss"]
+})
+export class GetNotifiedConfirmationModalComponent {
```
Comment (lpboucher) [inline src/app/modules/products/components/get-notified-confirmation-modal/get-notified-confirmation-modal.component.ts]: not a big fan of this component name. We should change the name of the component to NotificationModal.

Also, we should change the file names accordingly

**Review by lpboucher — CHANGES_REQUESTED** [review]

**Review by lpboucher — APPROVED** [review]

**Pull request merged**